packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -109,43 +124,68 @@ export const CatalogPanel: FC<React.PropsWithChildren<Props>> = ({
                 'ArrowDown',
                 () => {
                     //FIXME fix bug when multiple "fields" have the same name in search and you can't move
-                    if (selectedTable) {
+                    if (selection) {
                         //TODO move around grouped items
```

Comment:
nice, this is done then, right ?

Reply:
Not quite. I changed the selector type to do it, but didn't do work to make sure it was right yet. Im happy to do that next.

Reply:
I made this an issue cause it might as well be done along with the fields #10202